requirements.txt

```diff
@@ -0,0 +1,5 @@
+canonicalwebteam.flask-base==0.1.0
+Flask==1.0.2
+python-frontmatter==0.4.5
```

Comment:
Keep canonicalwebteam.templatefinder==0.1.1, remove bleach mistune and frontmatter.